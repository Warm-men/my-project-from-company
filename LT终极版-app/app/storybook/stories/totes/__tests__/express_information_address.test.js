import React from 'react'
import Address from '../express_information_address'
import render from 'react-test-renderer'

describe('express_information page', () => {
  describe('test render', () => {
    it('express_information', () => {
      const tree = render
        .create(
          <Address
            address={{
              address_1: '123',
              address_2: '321',
              city: '北京市',
              company: null,
              country: 'CN',
              customer_id: 665640,
              district: '东城区',
              full_name: '何兆',
              id: 21,
              state: '北京',
              telephone: '15555555555',
              zip_code: '100011'
            }}
          />
        )
        .toJSON()
      expect(tree).toMatchSnapshot()
    })
  })
})
