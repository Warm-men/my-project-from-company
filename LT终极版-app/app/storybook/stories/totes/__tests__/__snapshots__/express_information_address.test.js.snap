// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`express_information page test render express_information 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
      "marginLeft": 8,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#e85c40",
        "borderRadius": 20,
        "height": 20,
        "justifyContent": "center",
        "width": 20,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#fff",
          "fontSize": 11,
        }
      }
    >
      收
    </Text>
  </View>
  <Text
    style={
      Object {
        "color": "#333",
        "flex": 1,
        "fontSize": 14,
        "lineHeight": 20,
        "marginLeft": 16,
      }
    }
  >
    [收货地址] 
    北京
    北京市
    东城区
    123
     
    何兆
     
    15555555555
  </Text>
</View>
`;
